pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2,7 +2,17 @@
 
 with import ./lib.nix { inherit pkgs; };
 
-self: super: {
+self: super:
+
+let
+  # apply an override to a package if it’s build with GHC <8.0
+  ifLT8 = override: drv:
```

Comment:
This function should be in `lib.nix` and it should have a more descriptive name than `ifLT8`.